tools/json/sea_picro.json

```diff
@@ -0,0 +1,56 @@
+{
+  "build": {
+    "arduino": {
+      "earlephilhower": {
+        "boot2_source": "boot2_w25q64jv_4_padded_checksum.S",
+        "usb_vid": "0x2E8A",
+        "usb_pid": "0xF00A"
+      }
+    },
+    "core": "earlephilhower",
+    "cpu": "cortex-m0plus",
+    "extra_flags": "-D ARDUINO_SEA_PICRO -DARDUINO_ARCH_RP2040 -DUSBD_MAX_POWER_MA=500",
+    "f_cpu": "133000000L",
+    "hwids": [
+      [
+        "0x2E8A",
+        "0x00C0"
+      ],
+      [
+        "0x2E8A",
+        "0xF00A"
+      ]
+    ],
+    "mcu": "rp2040",
+    "variant": "sea_picro"
+  },
+  "debug": {
+    "jlink_device": "RP2040_M0_0",
+    "openocd_target": "rp2040.cfg",
+    "svd_path": "rp2040.svd"
+  },
+  "frameworks": [
+    "arduino"
+  ],
+  "name": "RP2040",
```

Comment:
Some how this ended up as `RP2040` as the board name although it should be `Sea-Picro`? Not sure what's happening in the pyhton script right there

Reply:
I got vendor and board name confused, I'm using "Generic" for the vendor since this is OSHW and the creator does not seem to have a company.